JS Basics/intro.js

```diff
@@ -0,0 +1,20 @@
+let sentence = "I can eat bananas all day";
```

Comment:
Just an advice, no need to change: whenever you are sure that your variable is immutable, it's more preferable to use `const` keyword for declaration according to JS best practices